pom.xml

```diff
@@ -2055,7 +2055,7 @@
     <properties>
 
         <!--Carbon Identity Framework Version-->
-        <carbon.identity.framework.version>5.20.137</carbon.identity.framework.version>
+        <carbon.identity.framework.version>5.20.138</carbon.identity.framework.version>
```

Comment:
Version number to be confirmed after merging wso2/carbon-identity-framework#3639